include/igraph_spatial.h

```diff
@@ -0,0 +1,44 @@
+/* -*- mode: C -*-  */
+/*
+   IGraph library.
+   Copyright (C) 2009-2012  Gabor Csardi <csardi.gabor@gmail.com>
+   334 Harvard street, Cambridge, MA 02139 USA
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc.,  51 Franklin Street, Fifth Floor, Boston, MA
+   02110-1301 USA
+
+*/
+#ifndef IGRAPH_SPATIAL
+#define IGRAPH_SPATIAL
+
+#include "igraph_datatype.h"
+#include "igraph_decls.h"
+#include "igraph_types.h"
+#include "igraph_error.h"
+#include "igraph_matrix.h"
+
+typedef enum {
+    IGRAPH_L2_METRIC
```

Comment:
Let's make this `IGRAPH_METRIC_L2`, which is more autocomplete friendly, and a more logical "most significant first" naming approach (though less grammatical in English).